app/src/main/java/com/radixdlt/android/RadixWalletApplication.kt

```diff
@@ -52,7 +52,11 @@ class RadixWalletApplication : Application(), HasActivityInjector, HasSupportFra
         AndroidThreeTen.init(this)
 
 //        connectToBootstrapNode()
-        RadixUniverse.bootstrap(Bootstrap.BETANET)
+//        RadixUniverse.bootstrap(Bootstrap.SUNSTONE)
```

Comment:
Maybe consider removing dead code.

Reply:
It's not really dead code, it's just that we have many different networks and things are always changing. At some point soon this will be reverted.